rest/route/project.go

```diff
@@ -235,6 +235,10 @@ func (h *projectIDPatchHandler) Parse(ctx context.Context, r *http.Request) erro
 	if err != nil {
 		return errors.Wrap(err, "error finding original project")
 	}
+	err = oldProject.MergeWithParserProject("")
```

Comment:
We don't want to do this; we use this to determine what the new project ref to be saved to the DB should be, so we'll end up saving project ref values.